pkg/cover/manager_to_ci.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package cover
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+)
+
+// CIDetails fields will be added to every CSV line.
+type CIDetails struct {
+	Timestamp      string `json:"timestamp"`
+	FuzzingMinutes int    `json:"fuzzing_minutes"`
+	Arch           string `json:"arch"`
+	Manager        string `json:"manager"`
+	KernelRepo     string `json:"kernel_repo"`
+	KernelBranch   string `json:"kernel_branch"`
+	KernelCommit   string `json:"kernel_commit"`
+	Compiler       string `json:"compiler"`
+}
+
+type dbCoverageRecort struct {
+	CIDetails
+	FilePath     string `json:"file_path"`
+	FunctionName string `json:"function_name"`
+	TotalCount   int    `json:"total_cb_count"`
+	CoveredCount int    `json:"covered_cb_count"`
```

Comment:
It will be the same table, but I wanted to land it later independently.
The idea - for every function we store:
1. hash(firstCoverFrame.Start(), lastCoverFrame.End(), allCoverFramesPositions). It enables us to correlate functions.
2. Cover points positions and their numbers.